test/flink_test.go

```diff
@@ -32,9 +33,14 @@ type flinkShellTest struct {
 	goldenFile string
 }
 
-func (s *CLITestSuite) TestFlinkArtifactList() {
+func (s *CLITestSuite) TestFlinkArtifact() {
 	tests := []CLITest{
+		{args: "flink artifact create my-flink-artifact --artifact-file test/fixtures/input/flink/java-udf-examples-3.0.jar", fixture: "flink/artifact/create.golden"},
```

Comment:
I think this test should still pass if the ".jar" file is an empty file? (There's no reason for it to be 10KB as far as I can tell)

Reply:
Yes, we use this for this system test repo too. https://github.com/confluentinc/cc-system-tests-scp-flink-udf/tree/master/resources

Reply:
"Yes" you agree, or "yes" we need it to be 10KB?

Reply:
This file is ok for testing. This jar file is simple and for helloworld function. As long as the file size not larger than 1G, we can create the artifact.